deploy/crds/hco00.crd.yaml

```diff
@@ -75,6 +101,25 @@ spec:
                     description: Enable HyperV strict host checking for HyperV enlightenments
                       Defaults to true, even when HyperConvergedFeatureGates is empty
                     type: boolean
+                  liveMigration:
+                    default: true
+                    description: Enables VMIs to be live migrated. Without this, migrations
+                      are not possible and will be blocked
+                    type: boolean
+                  sidecar:
+                    default: true
+                    description: Enables using our sidecar hooks for injecting custom
+                      logic into the VMI startup flow. This is a very advanced feature
+                      that has security implications, which is why it is opt-in only
```

Comment:
you say that it is opt-in, but the default is set to `true`. It does not make sense to me.
Can the description add a URL to an example of who this can be used?

Reply:
This one was explicitly enabled as for https://bugzilla.redhat.com/1745328 since 2.1.0.
I think that in this PR we should keep all the values exactly as they were in the past, then we should use a separate PR with a clear explanation each time we want to change a value and so the expected behaviour of the product.

Reply:
1. this PR should not be self-contradictory. It says that `sidecar` is opt-in, but does the opposite. This does not stand.
2. the BZ says nothing about the correct default value for this.
3. HCO never provided this API. Since it is so dangerous, I'm not at all convinced that it needs to be exposed at all. The `kubevirtconfg` annotation may be enough for this.
4. I'm OK with hard-coding this to `true`, but I prefer to have a follow-up commit fixing it to `false` included in this very PR. We don't want to expose our users to this any further moment.